offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/person/create/CreatePersonViewModel.groovy

```diff
@@ -51,10 +52,36 @@ class CreatePersonViewModel {
         this.customerService = customerService
         this.managerResourceService = managerResourceService
         this.personResourceService = personResourceService
-        availableAffiliations = new ObservableList(new ArrayList<Affiliation>(affiliationService.iterator().collect()))
+
+        List<Affiliation> affiliations = affiliationService.iterator().collect()
+        availableOrganisations = new ObservableList(new ArrayList<Organisation>(toOrganisation(affiliations)))
```

Comment:
Maybe Nit: I think seperating the conversion and the generation of an observable list makes it more understandable since you can distinguish between conversion and observation
```suggestion
        new ArrayList<Organisation> affiliationToOrganisation = toOrganisation(affiliations)
        availableOrganisations = new ObservableList(affiliationToOrganisation)
```